snap/snapcraft.yaml

```diff
@@ -0,0 +1,47 @@
+name: newsboat
+version: git
+summary: An RSS/Atom feed reader for text terminals
+description: |
+  Newsboat is a fork of Newsbeuter, an RSS/Atom feed reader for the text 
+  console. The only difference is that Newsboat is actively maintained 
+  while Newsbeuter isn't.
+grade: stable
+confinement: strict
+
+apps:
+  newsboat:
+    command: usr/local/bin/newsboat
+    environment:
+      LOCPATH: "$SNAP/usr/lib/locale"
+    plugs:
+      - network
+      - home
+  podboat:
+    command: usr/local/bin/podboat
+    aliases: [podboat]
+    environment:
+      LOCPATH: "$SNAP/usr/lib/locale"
+    plugs:
+      - network
+      - home
+
+parts:
+  newsboat:
+    plugin: make
+    source: .
+    build-packages: 
```

Comment:
Have a question: judging by [the docs](https://docs.snapcraft.io/build-snaps/syntax#build-packages), Snap auto-discovers dependencies using `ldd`. For example, `libsqlite3-dev` will pull in `libsqlite3`, and since the latter contains a library that Newsboat uses, `libsqlite3` package will be included in the final snap even though it's never explicitly mentioned anywhere here. Is that correct? Simply trying to wrap my head around dependency management here, so I can maintain it.

Reply:
Actually while that's the case now, snapcraft will soon be moving to a system where you explicitly list the stage packages, rather than have them guessed by snapcraft itself.

Reply:
BTW, is there some sort of newsletter yet? I've subscribed to forum.snapcraft.io as you suggested on the IRC when we discussed this before, but it's a firehose.